packages/x-codemod/src/util/renameIdentifiers.ts

```diff
@@ -129,7 +129,7 @@ export default function renameIdentifiers({
 }: RenameIdentifiersArgs) {
   root
     .find(j.Identifier)
-    .filter((path) => !!identifiers[path.node.name])
+    .filter((path) => Object.hasOwn(identifiers, path.node.name))
```

Comment:
What do you think about using [this new API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn#browser_compatibility) which causes to bump the node engine version requirement? 🤔 
- Leave it for the next major? (probably will not be updated to use this new API in that case 🙈 )
- Go with the node engine bump as the codemod is only used by developers anyways, and possibly very small amount of them are stuck only on `< v16.9.0` 🤔

Reply:
Ok ok... Nevermind. 🙈 
Our CI is strictly node 14 and until that is updated, it won't build the package. 🤦